src/steps/resource-manager/databases/sql/client.ts

```diff
@@ -215,13 +216,42 @@ export class SQLClient extends Client {
     }
   }
 
+  public async fetchServerVulnerabilityAssessments(
```

Comment:
nitpicky: this seems to return just one vuln assessment.
```suggestion
  public async fetchServerVulnerabilityAssessment(
```
```suggestion
  public async fetchServerVulnerabilityAssessments(
```

Reply:
Changed!